cppcache/include/geode/AttributesMutator.hpp

```diff
@@ -190,6 +191,22 @@ class APACHE_GEODE_EXPORT AttributesMutator {
    */
   void setCacheWriter(const std::string& libpath,
                       const std::string& factoryFuncName);
+
+  /** Sets partition resolver writer for region. The previous partition resolver
+   * will be replaced with <code>resolver</code>.
+   * @param resolver PartitionResolver
+   */
+  void setPartitionResolver(std::shared_ptr<PartitionResolver> resolver);
+
+  /** Sets cache writer for region. The previous partition resolver will be
+   * replaced with the one created using the factory function provided in
+   * the given library.
+   * @param library_path path of the library containing partition resolver
+   * factory function.
+   * @param function factory function for creating a partition resolver.
+   */
+  void setPartitionResolver(const std::string& library_path,
```

Comment:
Unless this is explicitly needed by your use case, and I really hope it isn't, let's avoid expanding on this API to load things from arbitrary libraries.

Reply:
Indeed, I don't actually need the library one. Good catch :) will remove it!